cartoframes/dataset.py

```diff
@@ -348,8 +349,8 @@ def _map_geom_type(self, geom_type):
     def _get_schema(self):
         if self.cc:
             return self.cc.get_default_schema()
-        else:
-            return 'public'
+
+        return None
```

Comment:
This `return None` is not required

Reply:
Yep, but I want to be explicit

Reply:
:+1:

Reply:
I like redundant and explicit return Nones :)